plugins/com.google.gcp.eclipse.appengine.localserver.test/build.properties

```diff
@@ -1,5 +1,5 @@
 source.. = src/
-output.. = bin/
+output.. = target/
```

Comment:
This should be `target/classes`, and it's needed in `plugins/com.google.gcp.eclipse.appengine.localserver` too.